proto/elys/assetprofile/tx.proto

```diff
@@ -11,57 +12,58 @@ service Msg {
   rpc CreateEntry (MsgCreateEntry) returns (MsgCreateEntryResponse);
   rpc UpdateEntry (MsgUpdateEntry) returns (MsgUpdateEntryResponse);
   rpc DeleteEntry (MsgDeleteEntry) returns (MsgDeleteEntryResponse);
+  rpc AddEntry    (MsgAddEntry   ) returns (MsgAddEntryResponse   );
 }
 message MsgCreateEntry {
```

Comment:
@fenriz07 can you remind me if we are keeping `MsgCreateEntry` as it is a duplicate of `MsgAddEntry` thanks

Reply:
Hi @cosmic-vagabond I was thinking test these changes on devne. if everything is ok  I-m going to delete these gov proposals after testing it

Reply:
@fenriz07 suggested to merge the PR as it is in order to test on devnet and later on create another PR for removal of the deprecated `CreateEntry` message